client/client.go

```diff
@@ -721,6 +737,32 @@ func (c *client) tryConnect(dispatcherCtx context.Context, dc string) (connectio
 	return connectionContext{}, err
 }
 
+// tryConnectToBatchProxy will create a stream to the TSO allocator follower to do the TSO requests batch to reduce
+// the pressure of its leader.
+func (c *client) tryConnectToBatchProxy(dispatcherCtx context.Context, dc string) (connectionContext, error) {
+	if followerClient, addr := c.followerClient(); followerClient != nil {
+		log.Info("use follower to forward tso stream to do the batch proxy", zap.String("dc", dc), zap.String("addr", addr))
+		forwardedHost, ok := c.getAllocatorLeaderAddrByDCLocation(dc)
```

Comment:
Need to consider removing the unreachable follower in time.

Reply:
Does `followerClinet()` only return healthy followerClient ?

Reply:
After reading the source code, it seems that `c.followerClient()` will only return the healthy follower connection.